monai/engines/workflow.py

```diff
@@ -162,7 +162,20 @@ def _register_post_transforms(self, posttrans: Callable):
 
         @self.on(IterationEvents.MODEL_COMPLETED)
         def run_post_transform(engine: Engine) -> None:
```

Comment:
could you make it a post transform handler? I feel in that way we could support arbitrary execution order...

Reply:
I see your point.
But I feel that is a big breaking change for current MONAI workflow, it will remove the `post_transform` arg of MONAI engines, need to update many tutorials or tests, and break all the NVIDIA Clara 4.0 products which are built based on current MONAI workflows API.
Maybe we can discuss within MONAI & Clara teams first then make a separate PR?
This PR is just an enhancement, will not break anything so far.
What do you think?

Thanks.

Reply:
I think `post_transform` is a wrong terminology here anyway... it should be postprocessing

Reply:
if it's a handler, we can attach it to engine by default so that it's non-breaking, what do you think?

Reply:
Hi @wyli ,

Or maybe we can keep current post transform logic and develop a new handler called `PostTransform` or `PostProcessing`, users can still use current post transforms in MONAI engines directly, or set it to `None` and use the new `PostTransform` handler. Or even use both.
What do you think?

Thanks.

Reply:
sounds good to me, thanks!

Reply:
Thanks, I created a separate ticket to track that feature: https://github.com/Project-MONAI/MONAI/issues/2223, will make a PR for it when our handler discussion with ignite team is ready.
And we may also add `MetricHandler` for current MONAI metrics, because ignite metrics are handlers in theory, but it's more complicated to achieve and based on ignite `Metric` base class. We can discuss it later.

About the `post transform` naming, some other DL papers or docs also use it:
https://www.sciencedirect.com/science/article/abs/pii/S1568494620309261
https://www.khronos.org/opengl/wiki/Post_Transform_Cache
Maybe @ericspod @rijobro can help share more comments, I am not sure about it.

Thanks.

Reply:
in that context, probably post-transform means some operations applied after the transforms. in our case, it should be some operations applied after the model forward, in our case it is "post_model_forward_transform"